### ISSUE TITLE
Openapi 3.0.0 path object items are not supported & openapi 3.0.* versions not supported

### ISSUE THREAD

**Issue by tdkmalan** [wso2/product-apim#3873]
**Description:**
1) Adding Path Item Objects to the OpenAPI 3.0.0 definition like "summary", "description", and specification extensions are not supported.
Getting the error - "The HTTP method 'summary' provided for the resource  is invalid"

2) Lates OpenAPI definition - Version 3.0.2 is not supported.

**Suggested Labels:**
APIM 2.5.0

**Affected Product Version:**
APIM 2.5.0

**Steps to reproduce:**
Create API using below definition
Ex1 - 

```
openapi: 3.0.0
info:
  title: sample
  version: 1.0.0
paths:
  /dummy:
    summary: Valid summary but invalid in WSO2
    description: Valid description but invalid in WSO2
    x-custom-field: Valid custom field but invalid in WSO2
    get:
      responses:
        200:
          description: OK

```
Ex2 - 

```
openapi: 3.0.2
info:
  title: sample
  version: 1.0.0
paths:
  /dummy:
    get:
      responses:
        200:
          description: OK
```

**Comment by tmkasun** [comment]
@praminda  Can we close this issue now ?

**Comment by praminda** [comment]
Yes we can close this now.

**Comment by dushaniw** [comment]
Reopening as we encounter the same issue in latest wum updated 2.6 pack

**Comment by dushaniw** [comment]
APIM 2.5 does not allow 'servers' elements in PathItem Object. This needs to be skipped when building URI Templates using the openAPI definition. This is already fixed in 3.0.0 branch with https://github.com/wso2/carbon-apimgt/pull/5842/commits/6f33a8c8a79f22bbb7a11a2e542422f324cf933e.

**Comment by dushaniw** [comment]
Fixed in latest wum updated 2.6
